src/applications/mhv/medications/components/shared/ExtraDetails.jsx

```diff
@@ -50,6 +50,13 @@ const ExtraDetails = rx => {
           updates.
         </p>
       )}
+      {dispStatus === dispStatusObj.activeParked && (
```

Comment:
Ideally, I think I would like to see a helper method With a clear name used in cases like these. (Maybe something like 
`prescriptionIsParked`)